src/Database/Schema/SchemaManager.php

```diff
@@ -62,7 +62,7 @@ public static function listTableDetails($tableName)
 
         $indexes = static::manager()->listTableIndexes($tableName);
 
-        return new Table($tableName, $columns, $indexes, $foreignKeys, false, []);
+        return new Table($tableName, $columns, $indexes, [], $foreignKeys, []);
```

Comment:
✅ seems to match the l10 branch on upstream repo: https://github.com/thedevdojo/voyager/blob/de0fd215ef67456c0c1304c37866c967db1bd1d0/src/Database/Schema/SchemaManager.php#L65